### PR TITLE
Residents Header and Councilor Stamp

### PR DESCRIPTION
**Pull request by aarmax**
Updated residents header images for rotation and

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -39,6 +39,15 @@
...
+#header-residents.header-residents-1 {
+	background-image: url("../img/header_residents1.png");
+}
+#header-residents.header-residents-2 {
+	background-image: url("../img/header_residents2.png");
+}
+#header-residents.header-residents-3 {
+	background-image: url("../img/header_residents3.png");
+}
 #header-data {
```
Comment (MinoMakesGames) [inline css/layout.css:51]: Checked against [https://github.com/jediholo/jedi-static/commit/3d2faf0d63171725c78b71ac9751790dbcd61ca4](this commit) and it looks good to me!

**Review by MinoMakesGames — APPROVED** [review]
Nice clean new images! Ship it :shipit:

**Comment by fcrespel** [comment]
@aarmax 2 remaining things:
1. the header_residents.png file still needs to exist (it can be the same as the "1" variant) as it loads before the Javascript code picks the random background
2. the CSS needs to be minimized with the scripts/minify.sh script, let me know if you can't run it on your side (it needs bash and java to be available)

**Review by fcrespel — APPROVED** [review]
LGTM!

**Pull request merged**